src/cfnlint/core.py

```diff
@@ -39,7 +40,7 @@ class UnexpectedRuleException(CfnLintExitException):
     """When processing a rule fails in an unexpected way"""
 
 
-def run_cli(filename, template, rules, regions, override_spec, build_graph, mandatory_rules=None):
+def run_cli(filename, template, rules, regions, override_spec, build_graph, registry_schemas, mandatory_rules=None):
```

Comment:
outside packages shouldn't be using run_cli by any concerns with the ordering here?  I've been adding parameters and setting a default value to keep some backwards compatibility.

Reply:
only found this from searching: https://github.com/Cloudzero/samwise/blob/793463e0774f9d1c702f4252377f19fa3408a558/samwise/__main__.py#L129 which looks already broken from https://github.com/aws-cloudformation/cfn-python-lint/pull/1411/
cc @silvexis

Reply:
have been generally contemplating compatibility for reliance on [undocumented internal functions/variables](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/docs/getting_started/integration.md)